src/constants/indicators.js

```diff
@@ -484,8 +484,8 @@ export default {
     source: 'Population-weighted density, Persons per square kilometre',
   },
   dependencyRatioNotWorkingAge: {
-    name: 'Population share not of working age',
-    format: FORMATS.PERCENT_ZERO_OR_ONE_DECIMAL,
+    name: 'Dependency ratio',
+    format: FORMATS.ONE_OR_TWO_DECIMALS,
```

Comment:
This seems odd. This indicator shows the proportion of (under 14 + over 65) / everyone else, and is _not_ a percentage. But the previous chart on the screen shows the same data, but shows each age group a proportion of the total and that _is_ a percentage?

They logically have to be the same units, right? Or am I missing something?

(top right and bottom left)

![image](https://user-images.githubusercontent.com/4443482/33148526-62cd1bc6-d020-11e7-82fb-16e8e4480180.png)